client/scripts/sw-docker-entrypoint

```diff
@@ -111,19 +111,28 @@ welcome() {
     echo "===================================="
 }
 
-main() {
+eval_task_prepare(){
     welcome $1
     pre_config
     pre_check
     set_python
     set_pip_cache
     restore_activate_runtime
+}
 
-    if [ "$1" = "cmp" ]; then
+case "$1" in
+    pre_config)
+        pre_config
+        ;;
+    cmp)
+        eval_task_prepare $1;
```

Comment:
Is it better to change the semicolon to `&& \`?

Reply:
done